NDTensors/src/NDTensors.jl

```diff
@@ -41,32 +41,43 @@ include("aliasstyle.jl")
 include("abstractarray/set_types.jl")
 include("abstractarray/to_shape.jl")
 include("abstractarray/similar.jl")
+include("abstractarray/ndims.jl")
+include("abstractarray/fill.jl")
 include("array/set_types.jl")
 include("tupletools.jl")
-include("dims.jl")
-include("tensorstorage.jl")
-include("default_storage.jl")
+include("tensorstorage/tensorstorage.jl")
+include("tensorstorage/default_storage.jl")
 include("tensorstorage/similar.jl")
-include("tensor.jl")
+include("tensor/tensor.jl")
+include("dims.jl")
 include("tensor/set_types.jl")
 include("tensor/similar.jl")
 include("adapt.jl")
-include("generic_tensor_operations.jl")
-include("contraction_logic.jl")
+include("linearalgebra/generic_tensor_operations.jl")
+include("linearalgebra/contraction_logic.jl")
 
 #####################################
 # DenseTensor and DiagTensor
 #
 include("dense/dense.jl")
-#include("dense/adapt.jl")
-include("fill.jl")
-include("symmetric.jl")
-include("linearalgebra.jl")
+include("dense/densetensor.jl")
+include("dense/TensorAlgebra/contract.jl")
```

Comment:
Should be lower case, i.e. `tensoralgebra`.

Reply:
Thanks!